pkg/network/tracer/event.go

```diff
@@ -376,6 +386,32 @@ func newIPRouteDest(source, dest util.Address, netns uint32) *ipRouteDest {
 	return d
 }
 
+func getPortType(p uint16) network.EphemeralPortType {
+	if !ephemeralChecked {
+		procfsPath := "/proc"
+		if config.Datadog.IsSet("procfs_path") {
+			procfsPath = config.Datadog.GetString("procfs_path")
+		}
+
+		intpair := sysctl.NewIntPair(procfsPath, "net/ipv4/ip_local_port_range", 0)
```

Comment:
The sysctl objects already have the concept of caching, so maybe we should create the `intpair` only once, but query it every time, rather than only checking it once. This would enable the value to be changed on the system and eventually be picked by system-probe without a restart.